.github/workflows/ci.yml

```diff
@@ -112,14 +117,28 @@ jobs:
             -p 9000:9000 \
             --entrypoint "./entrypoint.sh" \
             --volume ./.devcontainer/minio-entrypoint.sh:/entrypoint.sh \
-            --name miniocontainer \
             minio/minio
 
           while ! curl $AWS_ENDPOINT_URL; do
             echo "Waiting for $AWS_ENDPOINT_URL..."
             sleep 1
           done
 
+      - name: Start Azurite for Azure Blob Storage emulator tests
+        run: |
+          docker run -d \
+            --env-file .devcontainer/.env \
+            -p 10000:10000 \
+            mcr.microsoft.com/azure-storage/azurite
+
+          while ! curl $AZURE_STORAGE_ENDPOINT; do
+            echo "Waiting for $AZURE_STORAGE_ENDPOINT..."
+            sleep 1
+          done
+
+          # create container
+          az storage container create -n $AZURE_TEST_CONTAINER_NAME --connection-string $AZURE_STORAGE_CONNECTION_STRING
```

Comment:
cannot easily create azurite test container via a docker entrypoint script (as we do for minio) since azurite image does not include az cli.